docs/nima/metrics/metrics-capable-components.adoc

```diff
@@ -0,0 +1,101 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Metrics-Capable Modules
+:description: Helidon Nima metrics-capable components
+:keywords: helidon, metrics, metrics-capable, microprofile, guide
+:intro-project-name: {h1-prefix}
+:chk: icon:check[]
+:x: icon:times[]
+:rootdir: {docdir}/../..
+
+include::{rootdir}/includes/nima.adoc[]
+
+include::{rootdir}/includes/metrics/metrics-capable-components.adoc[tags=preamble;all-beginning-text;writing-code-intro]
+
+[[writing_SE,SE Application Techniques]]
+==== Writing and Packaging a Metrics-capable _Helidon SE Application_
+Write your _SE application_ similarly, but do not use the `ComponentMetricsSettings`.
+Instead, build a `MetricsSettings` object from the configuration.
+
+.Example code to support disabling metrics usage in a component
+[source,java]
+----
+import io.helidon.config.Config;
+import io.helidon.metrics.api.MetricsSettings;
+import io.helidon.metrics.api.RegistryFactory;
+import io.helidon.reactive.webserver.WebServer;
```

Comment:
As before, the code examples will probably change to reflect Nima instead of reactive. Updating these doc pages will be part of the metrics work for Nima.